demo/Makefile

```diff
@@ -11,7 +11,7 @@ run: build
 build-snapshot:
 	docker build -t $(IMAGE):RELEASE .
 	mkdir -p snapshot-plugins
-	for p in $$(cat plugins.txt|perl -pe s/:.+//g; cat snapshot-only-plugins.txt); do echo looking for snapshot builds of $$p; for g in org/jenkins-ci/plugins org/jenkins-ci/plugins/workflow org/jenkins-ci/plugins/pipeline-stage-view; do if [ -f ~/.m2/repository/$$g/$$p/maven-metadata-local.xml ]; then cp -v $$(ls -1 ~/.m2/repository/$$g/$$p/*-SNAPSHOT/*.hpi | tail -1) snapshot-plugins/$$p.jpi; fi; done; done
+	for p in $$(cat snapshot-only-plugins.txt); do echo looking for snapshot builds of $$p; for g in org/jenkins-ci/plugins org/jenkins-ci/plugins/workflow org/jenkins-ci/plugins/pipeline-stage-view; do if [ -f ~/.m2/repository/$$g/$$p/maven-metadata-local.xml ]; then cp -v $$(ls -1 ~/.m2/repository/$$g/$$p/*-SNAPSHOT/*.hpi | tail -1) snapshot-plugins/$$p.jpi; fi; done; done
```

Comment:
This is not really related, but it was really annoying to have every snapshot in my local Maven repo deployed in the snapshot docker image (`make build-snapshot`). I would expect to have snapshots deployed *only* for the plugins I explicitly add in `snapshot-only-plugins.txt`.